src/transactions/ClawbackOpFrame.cpp

```diff
@@ -0,0 +1,108 @@
+// Copyright 2021 Stellar Development Foundation and contributors. Licensed
```

Comment:
forgot to update visual studio files

Reply:
Is there a process to do this on a Mac? Maybe I can use Visual Studio for Mac? In the past I would just apply the diff you produced.

Reply:
it's xml (and no, don't use the Mac version that is not compatible) - you just need to look for a file that lives next to it, and duplicate the relevant sections.

Reply:
I updated the core vcxproj files.